agent/config/beerocks_agent.conf.in

```diff
@@ -12,7 +12,6 @@ temp_path=@TMP_PATH@
 vendor=Intel
 model=prplMesh
 ucc_listener_port=8002 # 0 - disabled
-ucc_listener_slave_hostap_iface=wlan0
```

Comment:
commit message grammar - " will be move " should be "will move" or "will be moved"

Reply:
Also, this should have been done as part of the commit which removed the previous implementation, shouldn't it?

Reply:
fixed the grammar mistake.
The previous commit removed a completely different thing, so no.